qml/TabLayout.qml

```diff
@@ -46,12 +50,14 @@ Item {
 
     // initialize first element in the tab
     Component.onCompleted:{
-        var new_stack = stack_component.createObject(null, {"id": 0, "anchors.fill": "parent"})
+        var new_stack = stack_component.createObject(null, {"stack_id": 0, "anchors.fill": "parent"})
         stack_layout.children.push(new_stack)
         refresh_layout(current_)
     }
 
     ChartsLayout {
+        z: 1
```

Comment:
z attribute not needed (visible attribute already manages its visibility)

Reply:
The `visible` attribute is really needed to avoid this scenario. Removing the `z`s in both cases.
![image](https://github.com/eProsima/Fast-DDS-monitor/assets/120394830/ece5ed36-725b-4359-a627-d3a90889749b)